server/serializer/page_subscription.go

```diff
@@ -75,6 +75,14 @@ func (ps PageSubscription) GetConfluenceURL() string {
 	return ps.GetSubscription().BaseURL
 }
 
+func (ps PageSubscription) GetEvents() []string {
```

Comment:
```suggestion
func (ps *PageSubscription) GetEvents() []string {
```
Why aren't we using this here?

Reply:
all the previous was like this that's why done like this. Should I leave this as it is or do this in every function.?

Reply:
I think you should do this in every function.